docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -708,6 +708,22 @@ WITH (
 )
 AS SELECT name, comment, regionkey FROM tpch.tiny.nation;
 ```
+(delta-lake-shallow-clone)=
+#### Shallow clone
+
+The connector supports reading from shallow cloned tables. A shallow clone is a
+source table that has been copied to a target destination at a specific version.
+Any changes made to shallow clones affect only the clones themselves and not the
+source table.
+
+To read from a cloned table, use the [`SELECT`](/sql/select) statement:
```

Comment:
Remove this .. its meaningless since you can run all sorts of queries so this is more confusing than helping